cpp/FileTransferAgent.cpp

```diff
@@ -847,20 +847,20 @@ RemoteStorageRequestOutcome Snowflake::Client::FileTransferAgent::downloadSingle
      }
      catch (...) {
        std::string err = "Could not open file " + fileMetadata->destPath + " to downoad";
-       char* str_error = sf_strerror(errno);
+       char strerr_buf[1024];
```

Comment:
for the error buffer we set size to 1024 in many places - should we create a constant from it?

Reply:
done